.github/workflows/workflow.yml

```diff
@@ -152,8 +153,27 @@ jobs:
         run: tox -e check_api_docs
       - name: Check spelling
         run: tox -e spell_check
-
-
+  protolint:
+    continue-on-error: False
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+        with:
+          python-version: 3.6
+      - uses: actions/setup-go@master
+        with:
+          go-version: '^1.14.0'
+      - name: Install protolint (ubuntu-latest)
+        run: |
+          sudo apt-get update --fix-missing
+          sudo apt-get autoremove
+          sudo apt-get autoclean
+          make protolint_install
+      - name: Protolint check
+        run: |
+          make protolint
```

Comment:
What do you think in adding a testing environment in `tox.ini`? So we can call `tox -e protolint` and have uniformity with other test environments.

Reply:
I agree!